src/screens/Cart/index.tsx

```diff
@@ -1,10 +1,59 @@
-import { Box,Text } from "native-base";
+import { AddIcon, Box, Button, ChevronLeftIcon, Heading, HStack, Image, MinusIcon, Stack, Text, VStack } from "native-base";
+import { useState } from "react";
 
-export default function Card() {
+export default function Cart() {
+    const [quant, setQuant] = useState(1)
+    const [price, setPrice] = useState(2.03)
+    const [bDisable, setBDisable] = useState(false)
 
-    return(
-        <Box>
-            <Text>Card</Text>
-        </Box>
+    console.log(quant)
+    return (
+        <>
+            <Box>
+                <ChevronLeftIcon size="5" />
+            </Box>
+            <Heading fontSize="40">Cart</Heading>
+            <Box alignSelf="center">
+
+
+                <Box flexDirection="row" rounded="lg" overflow="hidden" borderWidth={2} borderColor="gray.200">
+
+
+                    <Image
+                        source={{
+                            uri: "https://wallpaperaccess.com/full/317501.jpg"
+                        }}
+                        alt="Alternate Text"
+                        size={120}
+                    />
+                    <VStack>
+                        <Box paddingX={4} paddingY={4}>
+                            <Heading size="md">Carlsberg Premium Beer</Heading>
+                            <Text fontSize="xs">375ml Can  |  5%</Text>
+                        </Box>
+
+                        <HStack>
+                            <Stack space="2.5" px="8">
+                                <Stack direction="row" mb="2.5" mt="1.5" space={3}>
+                                    <Button size="6" variant="subtle" onPress={() => quant <= 1 ? setBDisable(true) : setQuant(quant - 1)} isDisabled={bDisable}>
+                                        <MinusIcon />
+                                    </Button>
+                                    <Text fontSize="md">{quant}</Text>
+                                    <Button size="6" variant="subtle" onPress={() => {
+                                        setQuant(quant + 1);
+                                        setBDisable(false)
+                                    }}>
+                                        <AddIcon />
+                                    </Button>
+                                </Stack>
+                            </Stack>
+                            <Stack>
+                                <Text color="#46DE99" fontSize="xl" fontWeight="bold" >R$ {(price * quant).toFixed(2)}</Text>
```

Comment:
Aqui é só questão de legibilidade, seria melhor criar uma variável lá em cima ou então criar uma função, talvez num arquivo chamado  `utils` na raiz do projeto, pra gente usar em outros lugares. 

* criar pasta utils,, dentro dela criar um arquivo que vai ter varias funções globais, por exemplo essa.

```suggestion
                                <Text color="#46DE99" fontSize="xl" fontWeight="bold" >{formatMoney(price * quant)}</Text>
```

Reply:
A função formatMoney pode ser feita com a api do javascript  `Intl` é bem fácil de usar, ela já formata e adicionar o R$